client/client.h

```diff
@@ -40,8 +40,8 @@ enum class ReadMode {
 // message just published.
 struct Message {
   Message() = default;
-  Message(size_t len, const void *buf, int64_t ordinal, int64_t timestamp)
-      : length(len), buffer(buf), ordinal(ordinal), timestamp(timestamp) {}
+  Message(size_t length_, const void *buffer_, int64_t ordinal_, int64_t timestamp_)
```

Comment:
Trailing underscores are only for member variables (per google style).  What's the warning here?

Reply:
The warning is about the local variables shadowing the members:

```
external/_main~_repo_rules~subspace/client/client.h:43:48: warning: constructor parameter 'ordinal' shadows the field 'ordinal' of 'Message' [-Wshadow-field-in-constructor]
  Message(size_t len, const void *buf, int64_t ordinal, int64_t timestamp)
                                               ^
external/_main~_repo_rules~subspace/client/client.h:47:11: note: previous declaration is here
  int64_t ordinal = -1;         // Monotonic number of message.
          ^
```

I didn't want to change the member variable names, since that affects so much more. The parameters to the constructor just have to not be exactly the same names as the members, because that is error-prone, and not allowed by certain high-integrity guidelines (MISRA, Autosar, etc.).

If you prefer some other naming style for the parameters, obviously, I don't care.

Reply:
No problem with calling the parameters something different from the members, just don't use a trailing underscore.  I hate trailing underscores but I'm trying to follow the google style as I last knew it.

Reply:
Alright, done.